tests/test_url_handler.py

```diff
@@ -0,0 +1,696 @@
+        'You have successfully nominated {0}'.format(form_data['url_value']),
+        'You have successfully added the {0} "{1}" for {2}'.format(
+            attribute_names[0], value, form_data['url_value'])
+    ]
+
+    assert url_handler.add_metadata(project, form_data) == expected
+
+
+def test_add_metadata_nominator_does_not_exist_in_project():
+    project = factories.ProjectFactory()
+    form_data = {'nominator_email': 'someone@someplace.com'}
+
+    with pytest.raises(http.Http404):
+        url_handler.add_metadata(project, form_data)
+
+
+@pytest.mark.parametrize('url, expected', [
+    ('http://www.example.com', 'http://www.example.com'),
+    ('   http://www.example.com   ', 'http://www.example.com'),
+    ('https://www.example.com', 'http://www.example.com'),
+    ('http://http://www.example.com', 'http://www.example.com'),
+    ('http://www.example.com///', 'http://www.example.com')
+])
+def test_check_url(url, expected):
+    assert url_handler.check_url(url) == expected
+
+
+def test_get_nominator_when_nominator_exists():
+    nominator = factories.NominatorFactory()
+    form_data = {'nominator_email': nominator.nominator_email}
+    assert url_handler.get_nominator(form_data) == nominator
+
+
+def test_get_nominator_when_nominator_does_not_exist():
+    form_data = {
+        'nominator_email': 'somebody@somewhere.com',
+        'nominator_name': 'John Smith',
+        'nominator_institution': 'UNT'
+    }
+
+    assert len(models.Nominator.objects.all()) == 1
+    new_nominator = url_handler.get_nominator(form_data)
+    assert len(models.Nominator.objects.all()) == 2
+    for key in form_data.keys():
+        assert getattr(new_nominator, key) == form_data[key]
+
+
+def test_get_nominator_when_nominator_cannot_be_created():
+    form_data = {
+        'nominator_email': 'somebody@somewhere.com',
+        'nominator_name': None,
+        'nominator_institution': None
+    }
+    new_nominator = url_handler.get_nominator(form_data)
+
+    assert new_nominator is False
+
+
+@pytest.mark.parametrize('scope_value, expected', [
+    ('1', 'You have already declared {} as "In Scope"'),
+    ('0', 'You have already declared {} as "Out of Scope"')
+])
+def test_nominate_url_nomination_exists(scope_value, expected):
+    project = factories.ProjectFactory()
+    nominator = factories.NominatorFactory()
+    form_data = {'url_value': 'http://www.example.com'}
+    factories.NominatedURLFactory(
+        url_nominator=nominator,
+        url_project=project,
+        entity=form_data['url_value'],
+        value=scope_value
+    )
+
+    assert (url_handler.nominate_url(project, nominator, form_data, scope_value) ==
+            [expected.format(form_data['url_value'])])
+
+
+@pytest.mark.parametrize('scope_value, expected', [
+    ('1', 'You have successfully declared {} as "In Scope"'),
+    ('0', 'You have successfully declared {} as "Out of Scope"')
+])
+def test_nominate_url_nomination_modified(scope_value, expected):
```

Comment:
I feel like we can simplify this test to just check if `"In Scope"` and `"Out of Scope"` are present in the summary list.